### ISSUE TITLE
Not working on OpenVZ VPS (Docker not compatible)

### ISSUE THREAD

**Issue by simongomez95** [ab77/netflix-proxy#181]
Is there any way to set netflix-proxy up manually without using Docker?

**Comment by ab77** [comment]
The original recipe was written without Docker, documented here:
http://blog.belodedenko.me/2014/02/diy-clone-of-netflix-tunlr-vps.html

It was written at a different time, so it's heavily out of date. You'll have to cross reference with the the GitHub repository, but apart from that, it should work.

**Comment by lbdroid** [comment]
@simongomez95 check this issue comment for an easier solution to implement; https://github.com/ab77/netflix-proxy/issues/180#issuecomment-224166147

This is a technically superior solution since it doesn't mess with dns and proxies, but rather just uses intelligent routing.

**Comment by simongomez95** [comment]
@lbdroid sorry for asking, this may be a really stupid question but thet solution can only be implemented on a router running OpenWRT, right? No way I can do it on a regular VPS running Ubuntu, Debian or CentOS?

**Comment by lbdroid** [comment]
It doesn't specifically have to be OpenWRT, although my instructions are tailored to it. But it *does* have to be done on the local network. And of course, you will have to set up *some* kind of tunnel to a remote server (I use openvpn, but could suggest other options as well, like l2tp or ssh+autossh).

OpenWRT is a good option, because its easy to set up on *really cheap* hardware. But if your router can't run OpenWRT, you could do it from a raspberry pi or any other linux machine plugged into your network. Disable the dhcp server on your router, enable a dhcp server on the raspberry pi, create a tunnel from the rpi to the remote machine, and set your network's default route through the rpi. The rpi will direct traffic either direct through the router, or through the tunnel.

Basically, along with an IP address and DNS server address, the DHCP server also pushes out routing information. In particular, it will push a default route with an IP address that typically matches its own IP address. When any machine on the network wants to send a packet to any other machine anywhere, it goes through the routing table in order to figure out the "next" machine it should send the packet to in order to get there. If your router has an IP address of 192.168.1.1, it will probably send out a default route via 192.168.1.1. You add the rpi onto the network with another ip address, like 192.168.1.2, and have it push THAT out as a default route, but IT should have ITS OWN default route set to 192.168.1.1. Then that rpi gets to control the data flowing out of your network.

**Comment by ab77** [comment]
Closing, no action.
